docs/usage.rst

```diff
@@ -90,19 +90,36 @@ regular TestCases.  So you can do something like:
 
 And you don’t have to clean the database yourself.
 
-If you have `factories`_ you want to instantiate on a per-scenario basis,
-you can initialize them in ``environment.py`` like this:
+django_ready hook
+-----------------
+
+You can add a ``django_ready`` method in your ``environment.py`` file in case
+you want to make per-scenario changes inside a transaction.
+
+For example, if you have `factories`_ you want to instantiate on a per-scenario
+basis, you can initialize them in ``environment.py`` like this:
 
 .. code-block:: python
 
     from myapp.main.tests.factories import UserFactory, RandomContentFactory
 
 
-    def before_scenario(context, scenario):
+    def django_ready(context, scenario):
+        # This method is run inside the transaction
```

Comment:
Does this comment demonstrate that the function name doesn't explain it all? Or why do we have to explain that "now we're finally inside a Django test case"?

I'm not criticizing just trying to think aloud.